src/components/shared/LeftSidebar.tsx

```diff
@@ -11,9 +11,39 @@ const LeftSidebar = () => {
   const { user } = useUserContext();
   const { mutate: signOut, isSuccess } = useSignOutAccount();
   const { pathname } = useLocation();
+
   useEffect(() => {
     if (isSuccess) navigate(0);
-  }, [isSuccess]);
+  }, [isSuccess, navigate]);
+
+  const handleInviteClick = () => {
+    const currentURL = "https://opengram.vercel.app/sign-in";
+    navigator.clipboard.writeText(`Hey! I am inviting you to join this amazing Social Media Application: ${currentURL}`)
+      .then(() => {
+        const popup = document.createElement('div');
+        popup.textContent = "Invitation copied to clipboard!";
+        popup.style.cssText = `
```

Comment:
Hey we are using Tailwind css Can you please convert these changes?

Reply:
also, Can you fix the UI   : 

![image](https://github.com/Nishitbaria/OpenGrame/assets/85815172/891c8783-3e44-448c-8225-fb66789997e4)

Its not look Good 

take reference of this  Figma file : [Link](https://www.figma.com/file/y7OzDDuGVslwr9rOYDQ7i2/OpenGrame?type=design&node-id=0%3A1&mode=design&t=H6AOHJuuNXj4Xrnk-1)